command/server.go

```diff
@@ -1564,6 +1570,7 @@ CLUSTER_SYNTHESIS_COMPLETE:
 
 	// When the underlying storage is raft, kick off retry join if it was specified
 	// in the configuration
+	// TODO: Should we also support retry_join for ha_storage?
```

Comment:
What's the context and scope of tacking care of this TODO? Do we have an issue to track it?

Reply:
This has been changed to be a stretch goal for now since there's a way for the followers to join manually. I'll remove this before merging if we're not able to get it in before the cutoff date.